package_creator.py

```diff
@@ -75,10 +75,25 @@ def insert_image_to_yml(dir_name, package_path, yml_data, yml_text):
     return yml_text, found_img_path
 
 
-def insert_script_to_yml(package_path, script_type, yml_text, dir_name, yml_data):
+def get_code_file(package_path, script_type):
```

Comment:
i think we can import this function from vaildate_files_structure

Reply:
You are correct. had some problems with the import and didn't want to waste on this time. let's do these changes in a separate  branch